nixos/modules/services/monitoring/graphite.nix

```diff
@@ -489,7 +490,10 @@ in {
 
             touch ${dataDir}/db-created
 
-            chown -R graphite:graphite ${cfg.dataDir}
+            chown graphite:graphite ${cfg.dataDir}
+            chown graphite:graphite ${cfg.dataDir}/db-created
+            chown graphite:graphite ${cfg.dataDir}/whisper
+            chown -R graphite:graphite ${cfg.dataDir}/log
```

Comment:
I had no idea what `db-created` even is. It was previously being set to have graphite as its owner, so I left it that way. Looking over it again, it seems like just a part of the way that nixos manages graphite and is in no way touched by graphite itself. So, I'll go ahead and change that and move `touch` to be last.